packages/graph/migrations/postgres/migration/1656417312397_initial.ts

```diff
@@ -61,6 +61,7 @@ export async function up(pgm: MigrationBuilder): Promise<void> {
       },
       version_id: {
         type: "UUID",
+        onDelete: "CASCADE",
```

Comment:
Required for proper database cleanup

Reply:
I talked about this with @thehabbos007. We probably want the `CASCADE` logic, for now, otherwise, we have to track every single operation, while the database implementation should be an implementation detail.

Reply:
I think we'd want to write recursive delete logic rather than keeping track of all of the operations. But I agree we can leave it _for now_.